repos/system_upgrade/el8toel9/actors/checkarmbootloader/libraries/checkarmbootloader.py

```diff
@@ -0,0 +1,46 @@
+from leapp import reporting
+from leapp.libraries.common.config.architecture import ARCH_ARM64, matches_architecture
+from leapp.libraries.common.config.version import get_source_version, get_target_version
+from leapp.libraries.stdlib import api
+
+
+def _inhibit_upgrade():
+    title = 'Upgrade RHEL {} to RHEL {} not possible for ARM machines.'.format(
+        get_source_version(), get_target_version())
```

Comment:
nitpick; you can keep the original if you want to.

Officially the upgrade to RHEL 9.5 is supported only from RHEL 8.10, so from this point I would even completely strip the info from what system they are upgrading. I was also thinking to replace target version directly by 9.5 string, but as people will start soon to test upgrade 8.10 -> 9.6, let's keep it like this until we apply the proper fix - which we know, it must be delivered for 9.6 upgrades for sure. Once the fix is applied, this one actor will be removed.

```suggestion
    title = 'Cannot upgrade to RHEL {} on ARM machines.'.format(get_target_version())
```

Reply:
I will keep it as is. When executed as supposed to nothing happens. When this somehow gets run when we don't expect it to, we get more information immediately